client/webserver/site/src/js/mm.ts

```diff
@@ -754,10 +759,14 @@ class Bot extends BotMarket {
     if (maxBase < 0 || maxQuote < 0) {
       throw Error(`rebalance math doesn't work: ${JSON.stringify({ bProj, qProj, maxBase, maxQuote })}`)
     }
-    return {
-      minBaseTransfer: Math.round(baseConfig.transferFactor * maxBase),
-      minQuoteTransfer: Math.round(quoteConfig.transferFactor * maxQuote)
-    }
+    const cex = app().mmStatus.cexes[cexName]
+    const mkt = cex.markets[mktID]
```

Comment:
If the exchange was unable to connect, if you load the mmsettings page, there will be an error because `mkt` will be undefined.